helix-core/src/main/java/org/apache/helix/manager/zk/ZkCacheBaseDataAccessor.java

```diff
@@ -126,41 +117,9 @@ public ZkCacheBaseDataAccessor(String zkAddress, ZkSerializer serializer, String
   public ZkCacheBaseDataAccessor(String zkAddress, ZkSerializer serializer, String chrootPath,
```

Comment:
This one shall be deprecated as well. And I would suggest keeping the very original behavior. Only return realmaware client through the new builder method.

Reply:
I will deprecate this but I do not agree with that we should revert the behavior.

Reply:
This is already deprecated..